Annotato/Annotato/Views/Document/Edit/SelectionBox/SelectionBoxViewModel.swift

```diff
@@ -0,0 +1,55 @@
+import CoreGraphics
+import Foundation
+
+class SelectionBoxViewModel: ObservableObject {
+    private(set) var id: UUID
+    private var startPoint: CGPoint
+    @Published var endPoint: CGPoint
+
+    init(id: UUID, startPoint: CGPoint, endPoint: CGPoint?) {
+        self.id = id
+        self.startPoint = startPoint
+        self.endPoint = endPoint ?? startPoint
+    }
+
+    func hasExceededBounds(bounds: CGRect) -> Bool {
+        let hasExceededTop = minY < bounds.minY
+        let hasExceededBottom = maxY > bounds.maxY
+        let hasExceededLeft = minX < bounds.minX
+        let hasExceededRight = maxX > bounds.maxX
+        if hasExceededTop || hasExceededBottom || hasExceededLeft || hasExceededRight {
+            return true
+        }
+        return false
```

Comment:
```suggestion
       !bounds.contains(frame)
```

Can you change this in `AnnotationViewModel::hasExceededBounds` too? I think this is a lot cleaner and I forgot about this in the previous PR.

Reply:
Also, can this method be in the same extension as where the `frame` variable is declared too? I don't think it's a good idea if something in the main class declaration depends on an extension of the same class.

Please update for `AnnotationViewModel` too.

Reply:
Done